tests/test_exceptions.py

```diff
@@ -24,6 +24,15 @@ def test_raise_for_hyp3_status():
     exceptions._raise_for_hyp3_status(response)
 
 
+def test_service_unavailable_error():
+    response = Response()
+    response.status_code = 503
+    response._content = b'{ "detail" : "foo" }'
+    with pytest.raises(exceptions.ServiceUnavailableError) as e:
```

Comment:
We could use `match=` here instead of asserting against `str(e)`, I assume we didn't know about that feature when these tests were originally written. But I'm also fine with leaving it as-is for consistency with the other tests.